spec/features/user_registers_spec.rb

```diff
@@ -0,0 +1,77 @@
+require "spec_helper"
+
+feature "User registers", js: true, vcr: true do
+  background do
+    visit register_path
+  end
+
+  valid_card_number = "4242 4242 4242 4242"
+  declining_card_number = "4000 0000 0000 0002"
+  expired_card_number = "4000 0000 0000 0069"
```

Comment:
These variables should be loaded via `let` or `given` (where `given` is preferred DSL for feature specs).